kogito-usertasks-custom-lifecycle-springboot/README.md

```diff
@@ -12,9 +12,33 @@ This example shows
 
 * working with user tasks
 * four eye principle with user tasks
-	
-	
+
+* Diagram	
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
+	
+Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties.png"></p>
+
+* Diagram Properties
```

Comment:
should be under `Diagram Properties (bottom)` to be in sync with other docs

Reply:
Done @radtriste